tests/spaND.cpp

```diff
@@ -340,23 +347,30 @@ int main(int argc, char* argv[]) {
     {
         VectorXd x = VectorXd::Zero(N);
         VectorXd b = VectorXd::Random(N);
-        if(useCG) {            
+        if (useCG) {            
             timer cg0 = wctime();
+            // SparseMatrix<typename SpMat::Scalar, Eigen::RowMajor> A_rm(A);
+            // int iter = cg(A_rm, b, x, t, iterations, solver_tol, verb);
+            //
+            // ^ Riley note: I added the lines above since my understanding is
```

Comment:
Might be true, not sure (probably depends on A, and on many other things). If possible I'd prefer not to change this.